Emote for cyborgs

AsnDen opened the pull request:
## About the PR
Removes screaming action for cyborgs, replaces it with beep action (i think the beep sound is the best)

## Why / Balance
Screaming don't work for cyborgs.

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
- tweak: Replaces scream action with beep action for cyborgs.

UbaserB commented on Resources/Prototypes/Actions/borgs.yml:
```diff
@@ -10,3 +10,14 @@
+- type: entity
+  id: ActionBeep
+  name: Beep
+  description: Beep Bepep
```
bepep moment? lol...

AsnDen replied:
Whoops. No bepep for cyborgs 😈

superjj18 commented:
Beep just doesn’t have the same “oh shit” energy as scream, really wish we had a proper robot scream, like a robot human scream

lzk228 commented:
i also wanted to do that, but with buzz sound, it's more similar to scream mood

ps3moira commented:
Just use the scream sound effect for borgs from goon https://github.com/ParadiseSS13/Paradise/blob/04e65c74969fb2c7f96db0a903747c18b990bfea/sound/goonstation/voice/robot_scream.ogg

lzk228 commented:
upload media please

AsnDen commented:
> upload media please

done

lzk228 commented:
gonna be honest, it doesn't even sound like borg and it's even worse than the previous vox laugh

GoldenCan commented:
I like it, but i think it is slightly too long and perhaps a bit too loud.

superjj18 commented:
> I like it, but i think it is slightly too long and perhaps a bit too loud.

I agree with the length being like a second longer than is should

Barrett-S commented:
As long as volume is normalized, this is my favorite PR ever. Perfect borg scream, 10/10.

superjj18 commented:
Two weeks no review 😢

ScarKy0 commented:
Honestly if the sound effect was slightly faster and less loud it would be perfect

chromiumboy commented:
Sorry, can't say that I'm a fan of the sound effect

Pumkin69 commented:
sound is fun should be shorter and maybe a little tiny bit quieter but everything else is good.

superjj18 commented:
3 months and no review(aside from the merge conflict)

ArtisticRoomba reviewed (changes requested):
An art lead has said that the sound is too loud, could you normalize the sound and have it be in line with the other scream emotes for humanoid species?

This is a personal comment coming from me: the scream is also a bit long, I think that it should be trimmed a little bit, to the same length, but just a little bit longer than the current scream emote for species.

superjj18 commented:
Damn all it took for that review was 7 months 🥲

AsnDen commented:
I've found out that previous screaming sound was likely to made from `malescream_2.ogg`
So I've tried to copy that effect. (Almost fully I hope). So I've made 9 new screaming sounds for cyborgs.

sounds from 1 to 6 are modified male screaming sounds (from 1 to 6 respectively)
sound 7 is modified female 1 screaming sound
sound 8 is modified female 2 screaming sound
sound 9 is modified female 4 screaming sound

Wanted to add wilheml but it was unrecognizable with this sound effects

Have updated main comment. Sound examples (3 of 9) are in media section.

I don't know how to orginize an `attributions.yml` properly. I've just added the list of files I modified and copied everything from original screaming files attributions.

EmoGarbage404 reviewed (approved):
seems good to me
